docs/docs/integrations/tools/databricks.ipynb

```diff
@@ -54,11 +62,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "from langchain_community.tools.databricks import UCFunctionToolkit\n",
+    "from databricks.sdk import WorkspaceClient\n",
```

Comment:
We should be able to remove this import after the change in `tool.py` below is released